usr/src/lib/libcmdutils/common/taskq.c

```diff
@@ -79,17 +91,21 @@ again:	if ((t = tq->tq_freelist) != NULL && tq->tq_nalloc >= tq->tq_minalloc) {
 			 * immediately retry the allocation.
 			 */
 			tq->tq_maxalloc_wait++;
-			rv = cv_timedwait(&tq->tq_maxalloc_cv,
-			    &tq->tq_lock, ddi_get_lbolt() + hz);
+
+			ts.tv_sec = 1;
+			ts.tv_nsec = 0;
+			err = cond_reltimedwait(&tq->tq_maxalloc_cv,
+			    &tq->tq_lock, &ts);
+
 			tq->tq_maxalloc_wait--;
-			if (rv > 0)
+			if (err == 0)
 				goto again;		/* signaled */
 		}
-		mutex_exit(&tq->tq_lock);
+		VERIFY0(mutex_unlock(&tq->tq_lock));
```

Comment:
This does nothing unless you're actually creating an error check lock. At which point you can actually just use mutex_enter and mutex_exit which are actually userland functions. Given that it's already been changed to use VERIFY, that transition probably isn't worth it.